### PR TITLE
Hide the placement blueprint when cursor is outside the playfield

### PR DESCRIPTION
**Pull request by smoogipoo**
Simple `Show()/Hide()` for now, but can be tweaked later with effects.

### CODE REVIEW

File: osu.Game/Screens/Edit/Compose/Components/BlueprintContainer.cs [hunk 1]
```diff
@@ -117,19 +130,34 @@ protected override bool OnClick(ClickEvent e)
             return true;
         }
 
+        protected override void Update()
+        {
+            base.Update();
+
+            if (currentPlacement != null)
+            {
+                bool cursorInPlayfield = composer.RulesetContainer.Playfield.ReceivePositionalInputAt(inputManager.CurrentState.Mouse.Position);
```
Comment (peppy) [inline osu.Game/Screens/Edit/Compose/Components/BlueprintContainer.cs]: as discussed, too much chaining. make a local property on composer which does the rest of the chain (probably virtual so the consumer could customise it).****

**Review by peppy — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
The blueprint should hide when outside the area.

image

This feels weird/wrong.

**Comment by smoogipoo** [comment]
Hmm something must not be working right if that's the case, will look into it.

**Comment by peppy** [comment]
It might be my fault, sec.

**Review by peppy — APPROVED** [review]

**Review by peppy — CHANGES_REQUESTED** [review]
As stated

**Review by peppy — APPROVED** [review]

**Pull request merged**